src/test/kotlin/com/jetbrains/snakecharm/fixtures/SnakemakeResolveTestCase.kt

```diff
@@ -1,21 +1,12 @@
 package com.jetbrains.snakecharm.fixtures
 
-import com.intellij.openapi.util.Ref
 import com.intellij.openapi.vfs.ex.temp.TempFileSystem
 import com.intellij.psi.PsiElement
 import com.intellij.psi.PsiNamedElement
-import com.jetbrains.python.psi.LanguageLevel
-import com.jetbrains.python.psi.impl.PyBuiltinCache
 import com.jetbrains.snakecharm.SnakemakeTestCase
 
 abstract class SnakemakeResolveTestCase : SnakemakeTestCase() {
     companion object {
-        fun <T : PsiElement> assertResolveResult(element: PsiElement?,
-                                                 aClass: Class<T>,
-                                                 name: String): T {
-            return assertResolveResult(element, aClass, name, null)
-        }
-
         fun <T : PsiElement> assertResolveResult(element: PsiElement?,
                                                  aClass: Class<T>,
                                                  name: String,
```

Comment:
According to Kotline code style guidelines (https://kotlinlang.org/docs/reference/coding-conventions.html):
> As a general rule, avoid horizontal alignment of any kind. 

also see function formatting conventions https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting

So here better is to format like
```
        fun <T : PsiElement> assertResolveResult(
                element: PsiElement?,
                aClass: Class<T>,
                name: String,
                containingFilePath: String?
        ): T {
```

Reply:
done